integreat_cms/cms/models/events/recurrence_rule.py

```diff
@@ -167,7 +167,7 @@ def advance() -> Iterator[date]:
         i = 0
         end_date = self.recurrence_end_date or date.max
         while True:
-            for _ in advance():
+            for __ in advance():
```

Comment:
```suggestion
            for _recurrence in advance():
```
Just a personal preference, but even if the variable is unused, I'd like to indicate its purpose.